packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -18,6 +22,15 @@ export class <%= className %> extends <%= modelBaseClass %> {
   <%= key %><%if (!val.required) {%>?<% } %>: <%= val.tsType %>;
 
 <% }) -%>
+
+<% if(allowAdditionalProperties) { -%>
+  // Define well-known properties here
+
+
```

Comment:
Do we need two empty lines here? I think prettier will remove the second line anyways, won't it? Can we keep a single empty line in this template?